### PR TITLE
JENKINS-66921 fix

### PR DESCRIPTION
**Pull request by mulawamichal**
If the RUNNING (or FINISHED) node did not exist, it failed with below error:

> Conversion error: Invalid time "0" (hour must be two digits)

corrected default value for those items from 0 to '00:00:00'

https://issues.jenkins.io/browse/JENKINS-66921

tested manually by processing faulty valgrind reports (attached to above jira ticket) with https://www.freeformatter.com/xsl-transformer.html

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by nfalco79 — CHANGES_REQUESTED** [review]

**Comment by nfalco79** [comment]
> Ensure you have provided tests - that demonstrates feature works or fixes the issue

i do not see additional test case. Please create more cases simply uploading you example file in 
src/test/resources/org/jenkinsci/plugins/xunit/types/valgrind/testcase3 and testcase4

File: src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl [hunk 1]
```diff
@@ -88,8 +88,8 @@ THE SOFTWARE.
     </xsl:variable>
 
     <xsl:template match="/valgrindoutput">
-        <xsl:variable name="startTime" select="xunit:millis-from-time(xunit:if-empty(substring(status/state[text() = 'RUNNING']/../time, 4), 0))" />
-        <xsl:variable name="endTime" select="xunit:millis-from-time(xunit:if-empty(substring(status/state[text() = 'FINISHED']/../time, 4), 0))" />
+        <xsl:variable name="startTime" select="xunit:millis-from-time(xunit:if-empty(substring(status/state[text() = 'RUNNING']/../time, 4), '00:00:00'))" />
```
[the remark this answers is not in the document]
Comment (mulawamichal) [reply src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl:91]: cannot. apparently, this function wants 2 params:
https://github.com/jenkinsci/xunit-plugin/pull/94/checks?check_run_id=6349149827

[the remark this answers is not in the document]
Comment (nfalco79) [reply src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl:91]: I already tested in my local, millis-from-time want just one

[the remark this answers is not in the document]
Comment (mulawamichal) [reply src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl:91]: ?? tried it:
https://github.com/jenkinsci/xunit-plugin/pull/94/commits/cd5515e56390cbf3dd5a257db5e2af7a1426206f
it failed

[the remark this answers is not in the document]
Comment (nfalco79) [reply src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl:91]: Look this PR #95

**Comment by nfalco79** [comment]
> as i wrote previously - i tested this using the mentioned page, as there is not even a slightest mention in README of how to run tests in this project.

all maven project execute test with `mvn clean verify`

File: src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl (hunk 1 above)
[the remark this answers is not in the document]
Comment (mulawamichal) [reply src/main/resources/org/jenkinsci/plugins/xunit/types/valgrind-1.0-to-junit.xsl:91]: great to see it. i\ll close this one. just please use the "anonimized" version of the problematic xml file

**Comment by nfalco79** [comment]
merged manually, thanks for the PR

**Pull request closed without merging**